R/formatter.r

```diff
@@ -4,6 +4,7 @@
 #' @param x a numeric vector to format
 #' @return a function with single parameter x, a numeric vector, that
 #'   returns a character vector
+#' @include utils.r
```

Comment:
Why is this necessary?

Reply:
This is necessary because a bit below ([`R/formatter.r, line 104`](https://github.com/zeehio/scales/blob/d7ebafb42df63e353e4168c8b6d370c02a57c8a0/R/formatter.r#L104)) the function `dollar` is defined as follows:

```r
#' @export
#' @rdname dollar_format
dollar <- dollar_format()
```
So `dollar_format()` is called at build time and `dollar_format()` calls `force_all()`, defined in `utils.r`. If you would rather that I put the `@include utils.r` in the dollar definition below, feel free to tell me.

Reply:
I’d rather get rid of the dependency altogether somehow

Reply:
Thanks for the review, I have got rid of it in the last commit :-) : https://github.com/hadley/scales/pull/81/commits/98c42d06cba2398c336159df1a9137701e3affcf